src/main/java/uk/gov/hmcts/reform/civil/utils/DocmosisTemplateDataUtils.java

```diff
@@ -16,14 +16,18 @@
 import static java.util.Optional.ofNullable;
 import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.ONE_V_TWO_TWO_LEGAL_REP;
 import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.getMultiPartyScenario;
+import static uk.gov.hmcts.reform.civil.handler.tasks.BaseExternalTaskHandler.log;
 
 public class DocmosisTemplateDataUtils {
 
     public static final int CASE_NAME_LENGTH_TO_FIT_IN_DOCS = 37;
     public static final String REFERENCE_NOT_PROVIDED = "Not Provided";
     //TODO Need to confirm the case name logic
     public static final Function<CaseData, String> toCaseName = caseData -> {
-        String caseName = fetchApplicantName(caseData) + " vs " + fetchRespondentName(caseData);
+        String caseName = null;
+
+        caseName = fetchApplicantName(caseData) + " vs " + fetchRespondentName(caseData);
+
```

Comment:
```suggestion
    String caseName = fetchApplicantName(caseData) + " vs " + fetchRespondentName(caseData);

```
don't need to split here